bitnami/kafka/templates/kafka-metrics-deployment.yaml

```diff
@@ -71,6 +71,6 @@ spec:
         - name: kafka-exporter-certificates
           secret:
             secretName: {{ .Values.metrics.kafka.certificatesSecret }}
-            defaultMode: 256
+            defaultMode: 0444
```

Comment:
What about changing this to `0440`? In that case, `1001` still has access to the files as it belongs to the `root` group. That way, we are being more restrictive, what do you think?

Reply:
yes it's better